Changelog

```diff
@@ -189,6 +189,7 @@ Changelog for 1.4.41
 * Refactor filter pages for income statement and balance sheet (Erik H)
 * Follow-up to earlier compensation for '.' missing from @INC (Erik H, #2275)
 * Fix reversal of payments against non-eca-default AR/AP account (Erik H, #2558)
+* Fix goods/services search with AR/AP invoices (Chris T, #2938)
```

Comment:
For the 1.5 branch, I'd put this in 1.4.42 *and* 1.5.9 (because it was in both of them: explains users of 1.5.2 why it's not in there for them)